variables.tf

```diff
@@ -379,7 +379,7 @@ variable "allowed_spot_instance_types" {
 variable "enable_worker_groups_launch_template" {
   description = "Flag to enable Worker Group Launch Templates"
   type        = bool
-  default     = false
+  default     = true
```

Comment:
Why is this default changed?

Reply:
The default configuration with this flag set to false would not work. For now, I'm not sure how to configure the add-on with this default setting.